### ISSUE TITLE
Different font size and line breaks for some extra attempts

### ISSUE THREAD

**Issue by Laura-O** [thewca/tnoodle#497]
## Actual (problem) behavior
One set of scrambles I generated for Berlin Winter Cubing 2020 had both extra attempts written in smaller fonts and in one line.

## Expected behavior
All move sequences for one set of scrambles should have the same font size and have equal linebreaks. 

## Steps To Reproduce
Generate a lot of 3x3x3 Scrambles...

## Desktop (please complete the following information):
macOS, FF 72.01, TNoodle 0.15.1, Java Version 8 Update 241

## Sheet
[3x3x3 Cube Round 2 Scramble Set A.pdf](https://github.com/thewca/tnoodle/files/4083857/3x3x3.Cube.Round.2.Scramble.Set.A.pdf)

**Comment by suushiemaniac** [comment]
Thank you for reporting this interesting bug!

I am able to reproduce this behaviour on the current master branch by explicitly invoking the exact scrambles from the PDF you specified. We will report back to you if the cause and potential fix has been found :)

**Comment by suushiemaniac** [comment]
Note to devs -- Reproduce by invoking the following API call against a locally running version:

`POST http://localhost:2014/view/Scrambles%20for%20Berlin%20Winter%20Cubing%202020.pdf`

using the following array for `sheets`:
```json
[
    {
        "scrambles": [
            "R2 U2 B2 F2 L F2 L2 U2 F2 R D L D' L2 D2 R' F U' F'",
            "R2 B2 D2 R2 B2 D F2 L2 U R2 F L D2 U' L R' U2 F2 R F' L2",
            "B2 U' R2 U F2 L2 D B2 D2 L' R B' U R B U2 R' U2",
            "B2 D' R2 D2 R2 U B2 U R2 D' L2 R D' F2 U' B R F' R' D",
            "B2 F2 D' L2 U R2 B2 R2 F' D R' U' B D2 R B F2 L2 B'"
        ],
        "extraScrambles": [
            "B2 U2 L2 B2 L2 U2 F U2 F' U F2 D L B D R' D R D'",
            "U D' L2 D' B' U' R' D' F' D2 L2 U' D2 L2 U F2 B2"
        ],
        "scrambler": "333",
        "copies": 1,
        "title": "3x3x3 Cube Round 2 Scramble Set A",
        "fmc": false,
        "colorScheme": {
            "B": "0000ff",
            "R": "ff0000",
            "D": "ffff00",
            "U": "ffffff",
            "F": "00ff00",
            "L": "ff8000"
        },
        "totalAttempt": 0,
        "attempt": 0,
        "group": "A",
        "event": "333",
        "round": 1
    }
]
```

**Comment by campos20** [comment]
Fixed on #500
